common/test/acceptance/pages/lms/video/video.py

```diff
@@ -605,7 +605,7 @@ def select_transcript_format(self, transcript_format):
         button = self.q(css=button_selector).results[0]
 
         coord_y = button.location_once_scrolled_into_view['y']
-        self.browser.execute_script("window.scrollTo(0, {});".format(coord_y))
+        self.browser.execute_script("window.scrollTo(0, {});".format(coord_y + 200))
```

Comment:
Ah those are coordinates. Sorry I was just glancing. Still, hardcoding doesn't sound great...thanks for looking for another solution.